ports/libe57format/portfile.cmake

```
@@ -14,7 +14,10 @@ vcpkg_cmake_configure(
     OPTIONS
         -DE57_BUILD_TEST=OFF
         -DE57_BUILD_SHARED=${E57_BUILD_SHARED}
+        -D57_RELEASE_LTO=OFF

```

```suggestion
        -DE57_RELEASE_LTO=OFF
```

based on https://github.com/asmaloney/libE57Format/blob/9c864c2a9785a4ebbb0acd81eb8879eeb793ce27/CMakeLists.txt#L116

?

---

Ah damn, when you think you hit the wrong key, but you missed a letter (I first had `-E57_RELEASE_LTO` and thought I hit `E` instead of `D`)